### PR TITLE
DTSERWFOUR-410-update-vemo-transfer-method-icon icon changes

### PR DESCRIPTION
**Pull request by pdamodarannair**

### CODE REVIEW

**Review by aparthibanpaypal — DISMISSED** [review]

File: Common/Sources/Helper/HyperwalletIcon.swift [hunk 1]
```diff
@@ -44,7 +44,7 @@ public final class HyperwalletIcon {
             return .wire
 
         case HyperwalletTransferMethod.TransferMethodType.venmoAccount.rawValue:
-            return .venmo
+            return .moneygram
```
Comment (ddhar-hw) [inline Common/Sources/Helper/HyperwalletIcon.swift:47]: This is not correct. Why would moneygram and venmo have the same icon? Have you tested what icon you are getting for moneygram?

**Review by ddhar-hw — CHANGES_REQUESTED** [review]

File: Common/Sources/Helper/HyperwalletIcon.swift (hunk 1 above)
Comment (ddhar-hw) [reply Common/Sources/Helper/HyperwalletIcon.swift:47]: @pdamodarannair and @aparthibanpaypal kindly check with Brandon regarding this. The changes should be done in `HyperwalletIconContent` not here

Comment (pdamodarannair) [reply Common/Sources/Helper/HyperwalletIcon.swift]: Modified as per review comments.

**Review by ddhar-hw — APPROVED** [review]

**Comment by ddhar-hw** [comment]
@SaiNarendran please test the new changes

**Review by aparthibanpaypal — APPROVED** [review]

**Review by SaiNarendran — APPROVED** [review]

**Pull request merged**